data-avro/src/main/java/com/linkedin/data/avro/SchemaTranslator.java

```diff
@@ -53,6 +58,7 @@ public class SchemaTranslator
   private static final Logger log = LoggerFactory.getLogger(SchemaTranslator.class);
 
   public static final String DATA_PROPERTY = "com.linkedin.data";
+  public static final String TRANSLATED_FROM_SOURCE_OPTION = "schema.translated.from.src";
```

Comment:
will `translated.from` enough?

Reply:
Yes. any unique key will work.

I'll update to `translated.from`